CONTRIBUTING.md

```diff
@@ -3,6 +3,10 @@
 
 First of all, thank you so much! We greatly appreciate all contributions. However, before you spend time working on Sharetribe, please read over our guidelines:
 
+## Issue reports
+
+Please use [ISSUE_TEMPLATE.md](https://raw.githubusercontent.com/sharetribe/sharetribe/master/ISSUE_TEMPLATE.md) for the issue report.
+
```

Comment:
I might also add "Check if the issue is already reported", even though it may seem obvious. From Github's example:

> Check existing issues for your issue. Duplicating an issue is slower for both parties so search through open and closed issues to see if what you’re running into has been addressed already.